asset_dashboard/static/js/components/ProjectsTable.js

```diff
@@ -1,21 +1,17 @@
 import React from 'react'
 import ReactTable from './BaseTable'
 import projectColumns from './table_utils/projectColumns'
+import SubRow from './table_utils/SubRow'
 
 const ProjectsTable = ({ allProjects, onAddToPortfolio, searchInput }) => {
-  const onRowClick = ({ original }) => {
```

Comment:
same thing as [this comment](https://github.com/fpdcc/ccfp-asset-dashboard/pull/171/files#r856438891).